sootup.java.bytecode/src/main/java/sootup/java/bytecode/inputlocation/PathBasedAnalysisInputLocation.java

```diff
@@ -789,4 +796,20 @@ public List<String> retrieveServletClasses(String extractedWARPath) {
       return classesInXML;
     }
   }
+
+  /**
+   * Refers to the rt.jar from <=Java8 as an AnalysisInputLocation requires: JAVA_HOME to be set and
+   * expects the jar in the "lib/" subdirectory. If you need to include the rt.jar from a custom
+   * Location please make use of JavaClassPathAnalysisInputLocation.
+   */
+  private static class DefaultRTJarAnalysisInputLocation extends ArchiveBasedAnalysisInputLocation {
```

Comment:
1. shouldn't this be public? 
2. Also should we now switch to this everywhere we add rt.jar manually?